config/ttbar.yaml

```diff
@@ -64,4 +64,4 @@ defaults:
 - classifier: classifier
 - default
 - hydra
-- _self_
```

Comment:
You should get rid off this kind of changes, eg with 'git checkout main file' to just pull this file from the main branch. Same for the other config files.

Reply:
Solved.